core/src/main/java/org/apache/accumulo/core/client/impl/Tables.java

```diff
@@ -147,6 +118,7 @@ public static void clearCache(Instance instance) {
     cacheResetCount.incrementAndGet();
     getZooCache(instance).clear(ZooUtil.getRoot(instance) + Constants.ZTABLES);
     getZooCache(instance).clear(ZooUtil.getRoot(instance) + Constants.ZNAMESPACES);
+    tableMapCache = null;
```

Comment:
There is also a clearCacheByPath method where this should be set to null

Reply:
Good catch, thank you!